pkg/proxy/util/utils.go

```diff
@@ -78,6 +78,25 @@ func BuildPortsToEndpointsMap(endpoints *v1.Endpoints) map[string][]string {
 	return portsToEndpoints
 }
 
+// ContainsLoopback returns true if the input is empty or one of the CIDR contains a loopback address.
+func ContainsLoopback(cidrStrings []string) bool {
+	if len(cidrStrings) == 0 {
+		return true
+	}
+	for _, cidr := range cidrStrings {
+		if IsZeroCIDR(cidr) {
+			return true
+		}
+		// swallow the error, we are only interested in finding cidrs containing loopbacks
+		// The cidrs should have been validated before.
+		ip, _, _ := netutils.ParseCIDRSloppy(cidr)
+		if ip.IsLoopback() {
```

Comment:
if I pass 0.0.0.0/1 or 64.0.0.0/2 or 96.0.0.0/3 (etc), that won't trigger this.  Should it?

Reply:
actually this is IPv4 only, it should not check IPv6

Reply:
fixes, great catch